runbotics/runbotics-scheduler/src/scheduler-database/process-credential/process-credential.service.ts

```diff
@@ -77,6 +78,27 @@ export class ProcessCredentialService {
         );
     }
 
+    async update(processId: number, user: User, editDto: EditProcessCredentialArrayDto) {
```

Comment:
I would rename `editDto` to something more descriptive like `processCredentialsToUpdate`